lib/vsc/utils/generaloption.py

```diff
@@ -683,6 +683,7 @@ def _is_enable_disable(x):
                 opt._long_opts = [x for x in opt._long_opts if not _is_enable_disable(x)]
         return fh
 
+    # pylint: disable=arguments-differ
```

Comment:
@JensTimmerman Rather than ignoring it, can/should we fix it?

Reply:
no, this is by design, no way to fix this